src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Overlay/OverlayOperatorTests.cs

```diff
@@ -143,15 +109,16 @@ public void GivenAOCIFileLayer_WhenGenerateDiffOCIFileLayer_IfBaseLayerFolderIsE
         public void GivenAOCIFileLayer_WhenGenerateDiffOCIFileLayerWithSnapshot_ADiffOCIFileLayerShouldBeReturned()
         {
             var overlayFs = new OverlayFileSystem("TestData/UserFolder");
-            overlayFs.ClearImageLayerFolder();
+            overlayFs.ClearBaseLayerFolder();
             Directory.CreateDirectory("TestData/UserFolder/.image/base");
-            File.Copy("TestData/Snapshot/baselayer.tar.gz", "TestData/UserFolder/.image/base/layer1.tar.gz", true);
-            var fileLayer = overlayFs.ReadMergedOCIFileLayer();
-            var baseFileLayer = GenerateBaseFileLayer(overlayFs);
-            var diffLayers = _overlayOperator.GenerateDiffLayer(fileLayer, baseFileLayer);
-
-            File.Delete("TestData/UserFolder/.image/base/defaultlayer.tar.gz");
-            Assert.Equal(819, diffLayers.FileContent.Count());
+            File.Copy("TestData/TarGzFiles/layer1.tar.gz", "TestData/UserFolder/.image/base/layer1.tar.gz", true);
+            var fileLayer = overlayFs.ReadOCIFileLayer();
+            var baseLayers = overlayFs.ReadBaseLayer();
+            var baseOCIfileLayer = _overlayOperator.ExtractArtifactLayer(baseLayers);
+            var diffLayers = _overlayOperator.GenerateDiffLayer(fileLayer, baseOCIfileLayer);
+
+            overlayFs.ClearBaseLayerFolder();
+            Assert.Equal(839, diffLayers.FileContent.Count());
```

Comment:
In this scenario, we are actually comparing whether the byte length of artifact layer is equal to the file content length, can we read the file content and compare it with `diffLayers` instead of using a literal constant?

Reply:
By offline discussion, this value is not frequently changed and hard to programmatically calculated, so we can keep the literal value here, thanks!